code-challenges.test.js

```diff
@@ -28,16 +32,43 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+const num1 = 15 
 // Expected output: "15 is divisible by three"
 const num2 = 0
 // Expected output: "0 is divisible by three"
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
 
+describe("dividend", () => {
+    test("takes a number as an argument and decides if the number is evenly divisble by three or not")
+    const num1 = 15
+    const num2 = 0
+    const num3 = -7
+    expect(dividend(num1)).toEqual("num1 is divisible by 3")
+    expect(dividend(num2)).toEqual("num2 is divisible by 3")
+    expect(dividend(num3)).toEqual("num3 is not divisible by 3")
+})
+
+
 // b) Create the function that makes the test pass.
 
+//Pseudocode
+// Declared a function called dividend
+// used 3 parameters num1,num2,num3
+// used conditional statments for each variable using modulus to determine if divisible by 3
+
+
+const dividend = (num1, num2, num3) => {
+    if (num1 % 3 === 0) {
+        return "is divisible by 3" }
+        else if (num2 % 3 === 0){
+            return "is divisible by 3"
+        } else (num3 % 3 === 0) 
+                return "is not divisible by 3"
+            }
```

Comment:
You are heading in the right direction with the big picture logic of needing to evaluate using conditionals with modulo operator.  I want to encourage you to try working on making functions dynamic and reusable.  Since the main goal is to check only a single number at a time when running the function, you should only need one parameter that can be any variable.